xcdat/spatial.py

```diff
@@ -173,7 +173,7 @@ def average(
         Using custom weights for averaging:
 
         >>> # The shape of the weights must align with the data var.
-        >>> self.weights = xr.DataArray(
```

Comment:
This documentation update is more generic than this PR and should probably be implemented separately...